EventFilter/RPCRawToDigi/src/RPCAMCLinkEvents.cc

```diff
@@ -0,0 +1,76 @@
+#include "EventFilter/RPCRawToDigi/interface/RPCAMCLinkEvents.h"
+
+RPCAMCLinkEvents::RPCAMCLinkEvents()
+{}
+
+std::string RPCAMCLinkEvents::getEventName(unsigned int id)
+{
+    switch (id & (RPCAMCLinkEvent::event_mask_ | RPCAMCLinkEvent::group_mask_)) {
```

Comment:
``` (RPCAMCLinkEvent::event_mask_ | RPCAMCLinkEvent::group_mask_) ``` is repeated quite a few times.
any good reason to be so verbose and repetitive?

Reply:
@slava77 I think no, reducing repetitions with a constexpr now.